objectrocket/instances/mongodb.py

```diff
@@ -95,6 +99,17 @@ def shards(self, add_shard=False):
         return response.json()
 
     @property
+    @util.token_auto_auth
+    def new_relic_stats(self):
+        """Get stats for this instance.
+        """
+        if self._new_relic_stats is None:
+            self._new_relic_stats = json.loads(requests.get(self._stats_url,
+                                           **self._instances._default_request_kwargs
+                                           ).content)
+        return self._new_relic_stats
```

Comment:
Per my comment below, I would updated this code to build its own URL on the spot, like so:
```python
url = '{}{}'.format(self._url, 'new-relic-stats')
```